airflow/contrib/sensors/aws_sqs_sensor.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.sensors.base_sensor_operator import BaseSensorOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.contrib.hooks.aws_sqs_hook import SQSHook
+from airflow.exceptions import AirflowException
+
+
+class SQSSensor(BaseSensorOperator):
+    """
+    Get messages from an SQS queue and passes them through XCom.
+
+    """
+
+    template_fields = ['sqs_queue', 'max_messages']
+
+    @apply_defaults
+    def __init__(self, aws_conn_id, sqs_queue, max_messages=5, *args, **kwargs):
+        """
```

Comment:
This is not the correct syntax. You can not add documentation to the class and constructor at the same time. You must select only one option, otherwise one of the options will be ignored in the process of building documentation.